docs/Architecture/Architectural-Overview.md

```diff
@@ -148,13 +148,22 @@ Each component/sub-component directory should consist of an `.sls` (Salt state)
   * PIP
   * Custom build requirements from source tar
 
-* `config`: Any configurations required for base (factory/field) setup of the component shall be covered in this file/directory. In ideal world, this would be a simple call to component provided `init` file. This state file should also allow for correcting any deviations caused by user over-rides.
+* `config`: Any configurations required for base (factory/field) setup of the component shall be covered in this file/directory. In ideal world, this would be a simple call to component provided `init` file. This state file should also allow for correcting any deviations caused by user over-rides.\
+The config folder structure will look like this:
+```
+├───component
+│   ├───config
+│   │   ├───post_install
+│   │   ├───config
+│   │   ├───init_mod
+
+```
 
-  This would consist of:
+each file would consist of:
```

Comment:
A new line starts in Caps. Do maintain the basic grammar discipline.

`each file would consist of:` => `Each sub-level file would consist of calls to respective southbound API as:  `